### ISSUE TITLE
Dare I say... chess?

### ISSUE THREAD

**Issue by lazarchitect** [lazarchitect/FlaskReactor#43]

**Comment by lazarchitect** [comment]
'chess; table will need a winner column, much the same way tictactoe has.

**Comment by lazarchitect** [comment]
Two important preliminary pieces of work before we can even start coding the chess logic.
1. The chess board (formerly Board.js) should be converted into a functional component. That way, we can use hooks, and avoid unnecessary lines. This leads into my next point...
2. The chessboard needs to use state. We need to remember, and set, the pieces on the board.

**Comment by lazarchitect** [comment]
starting coding on branch "CHESS"

**Comment by lazarchitect** [comment]
To Do 
- client side JS logic handles onClick. Which tiles light up? 
- server side websocketHandler decides if the move is truly valid, and if so updates the client with the new boardstate and any other changes (check/mate, promotion)

**Comment by lazarchitect** [comment]
blocked by #69

**Comment by lazarchitect** [comment]
#69 complete, Blocked tag removed

**Comment by lazarchitect** [comment]
Closing ticket just because it's super old and partially complete; the initial scope was probably way too wide for a single ticket. It's also just poorly documented. I think this was originally to handle the coding of the chess piece movement logic?

I'm going to open a new ticket with a more clearly defined goal and tasks.
